[2023-04-06T07:30:14.069+0000] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: io_resto_casa.respond scheduled__2023-04-06T07:20:00+00:00 [queued]>
[2023-04-06T07:30:14.092+0000] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: io_resto_casa.respond scheduled__2023-04-06T07:20:00+00:00 [queued]>
[2023-04-06T07:30:14.094+0000] {taskinstance.py:1288} INFO - 
--------------------------------------------------------------------------------
[2023-04-06T07:30:14.096+0000] {taskinstance.py:1289} INFO - Starting attempt 1 of 3
[2023-04-06T07:30:14.097+0000] {taskinstance.py:1290} INFO - 
--------------------------------------------------------------------------------
[2023-04-06T07:30:14.124+0000] {taskinstance.py:1309} INFO - Executing <Task(PythonOperator): respond> on 2023-04-06 07:20:00+00:00
[2023-04-06T07:30:14.136+0000] {standard_task_runner.py:55} INFO - Started process 6757 to run task
[2023-04-06T07:30:14.142+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'io_resto_casa', 'respond', 'scheduled__2023-04-06T07:20:00+00:00', '--job-id', '142', '--raw', '--subdir', 'DAGS_FOLDER/ciao.py', '--cfg-path', '/tmp/tmp14kc6kiz']
[2023-04-06T07:30:14.146+0000] {standard_task_runner.py:83} INFO - Job 142: Subtask respond
[2023-04-06T07:30:14.272+0000] {task_command.py:389} INFO - Running <TaskInstance: io_resto_casa.respond scheduled__2023-04-06T07:20:00+00:00 [running]> on host cfece00ff772
[2023-04-06T07:30:14.412+0000] {taskinstance.py:1518} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=io_resto_casa
AIRFLOW_CTX_TASK_ID=respond
AIRFLOW_CTX_EXECUTION_DATE=2023-04-06T07:20:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-04-06T07:20:00+00:00
[2023-04-06T07:30:14.415+0000] {python.py:177} INFO - Done. Returned value was: Greet Responded Again
[2023-04-06T07:30:14.464+0000] {taskinstance.py:1332} INFO - Marking task as SUCCESS. dag_id=io_resto_casa, task_id=respond, execution_date=20230406T072000, start_date=20230406T073014, end_date=20230406T073014
[2023-04-06T07:30:14.516+0000] {local_task_job.py:212} INFO - Task exited with return code 0
[2023-04-06T07:30:14.545+0000] {taskinstance.py:2596} INFO - 0 downstream tasks scheduled from follow-on schedule check
